XLIFF support (#35)
intento/intento-api: ai.text.translate.md

### Supported formats in async mode

In addition to the provider's format support, we have built-in support for:
 - HTML/XML

>  - XLIFF

Reviewer: ```suggestion
for HTML, XML and XLIFF formats.```

#### XLIFF format

#### Memsource version of the XLIFF

We support the [Memsource version of the XLIFF (v1.2)](https://help.memsource.com/hc/en-us/articles/360004658951) format in [async mode](https://github.com/intento/intento-api#async-mode). To translate Memsource version of the XLIFF add `format`: `xliff-memsource`:

```sh
curl -XPOST -H 'apikey: YOUR_API_KEY' 'https://api.inten.to/ai/text/translate' -d '{
    "context": {
        "text": "<xliff version=\"1.2\" xmlns....",
        "format": "xliff-memsource"
    },
    "service": {
        "provider": "ai.text.translate.microsoft.translator_text_api.3-0",
        "async": true
    }
}'
```

> `From` and `to` parameters are specified in the XLIFF file as [source-language](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#source-language) tag and [target-language](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#target-language) tag, respectively. We also support the translation of the XLIFF format from the [console](https://console.inten.to).

Reviewer 2: `from` and `to` parameters are specified in the XLIFF file as [source-language](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#source-language) tag and [target-language](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#target-language) tag respectively.

We also support translation of the Memsource xliff (mxliff) files in [Intento Console](https://console.inten.to).

#### XLIFF format

#### Memsource version of the XLIFF

We support the [Memsource version of the XLIFF (v1.2)](https://help.memsource.com/hc/en-us/articles/360004658951) format in [async mode](https://github.com/intento/intento-api#async-mode). To translate Memsource version of the XLIFF add `format`: `xliff-memsource`:

```sh
curl -XPOST -H 'apikey: YOUR_API_KEY' 'https://api.inten.to/ai/text/translate' -d '{
    "context": {
        "text": "<xliff version=\"1.2\" xmlns....",
        "format": "xliff-memsource"
    },
    "service": {
        "provider": "ai.text.translate.microsoft.translator_text_api.3-0",
        "async": true
    }
}'
```

> `from` and `to` parameters are specified in the XLIFF file as [source-language](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#source-language) tag and [target-language](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#target-language) tag respectively.

Reviewer: ```suggestion
`from` and `to` parameters are not required here because they are specified in the XLIFF file itself as [source-language](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#source-language) tag and [target-language](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#target-language) tag respectively.
```

> We also support translation of the Memsource XLIFF (MXLIFF) files in [Intento Console](https://console.inten.to).

Reviewer: ```suggestion
Try this functionality in the [Intento Web Tools](https://console.inten.to/translate/web-tools).
```
